trp_external_user/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "External users",
+    "summary": "Allow external users on your system",
+    "version": "13.0.1.0.0",
+    "author": "Therp BV, Sunflower IT",
+    "category": 'External users',
+    'website': 'http://therp.nl',
+    'depends': [
```

Comment:
Inconsistent quotation. Just black all of the code in the module. Best to install pre-commit to enforce this. The needed configuration is already in the 13.0 branch. See: https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97